pages/resultset.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.base import Base
+
+
+class ResultsetPage(Base):
+
+    _job_log_locator = (By.CSS_SELECTOR, '.logviewer-icon')
+    _job_resultpane_status = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span:nth-child(2)')
+    _log_viewer_locator = (By.CSS_SELECTOR, 'logviewer-icon')
+    _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
+    _result_status_locator = (By.ID, 'result-status-pane')
+
+    @property
+    def job_resultpane_status(self):
+        self.selenium.find_element(*self._result_status_locator)
+        return self.selenium.find_element(*self._job_resultpane_status).text
+
+    def go_to_page(self):
+        self.open('')
+
+    def open_next_failed_job(self):
```

Comment:
Probably best to call this `next_unclassified_failure` in keeping with our treeherder code base. The 'n' key will only find those types. After sheriffs classify them the 'n' key would not longer hit them, though they are also thought of as a failed job. I know.. subtleties abound in treeherder :)